metadata-ingestion/src/datahub/ingestion/source/aws/s3_util.py

```diff
@@ -1,104 +1,12 @@
 import logging
 import os
-from typing import Iterable, Optional
-
-from datahub.emitter.mce_builder import make_tag_urn
-from datahub.ingestion.api.common import PipelineContext
-from datahub.ingestion.source.aws.aws_common import AwsSourceConfig
-from datahub.metadata.schema_classes import GlobalTagsClass, TagAssociationClass
 
 S3_PREFIXES = ["s3://", "s3n://", "s3a://"]
 
 logging.getLogger("py4j").setLevel(logging.ERROR)
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-def get_s3_tags(
```

Comment:
Why is this removed ?

Reply:
They are actually moved to the other util class s3_boto_util as a part of another PR. Not sure why these changes are coming here. But i didnt removed the changes as these needs to go away

Reply:
Okay, ideally this should go away in final PR.